iotilecore/iotile/core/hw/auth/auth_chain.py

```diff
@@ -45,16 +45,19 @@ def _load_installed_providers(self):
             self._auth_factories[name] = entry
 
     def get_serialized_key(self, key_type, device_id, **key_info):
+        """Deligates call to auth providers in the chain
+        """
```

Comment:
docstrings that are only one line long should have closing quotes on the same line, sorry for not clarifying that (Need to pull it out of the internal doc and put it in the codebase)

Reply:
Not at all, thank you for pointing this out